src/DependencyInjection/DI.Specification.Tests/src/Fakes/ClassInheritingAbstractClass.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.DependencyInjection.Specification.Fakes
+{
+    public class ClassInheritingAbstractClass : AbstractClass
```

Comment:
I don't see ClassAlsoInheritingAbstractClass  and ClassInheritingAbstractClass used anywhere.